src/components/Contracts/index.tsx

```diff
@@ -0,0 +1,236 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+/* eslint-disable global-require */
+
+import React from 'react';
+import clsx from 'clsx';
+import Translate from '@docusaurus/Translate';
+import Heading from '@theme/Heading';
+
+const LSPs_AccountsInteraction = [
+  {
+    name: '👤 LSP0 ERC725 Account',
+    url: 'https://docs.lukso.tech/contracts/overview/UniversalProfile',
+    urlABI: 'https://docs.lukso.tech/contracts/contracts/LSP0ERC725Account',
+    description: (
+      <Translate id="playground.codesandbox.description">
+        A smart contract based account that can be used to represent an identity
+        on-chain.
+      </Translate>
+    ),
+  },
+  {
+    name: '📢 LSP1 Universal Receiver',
+    url: 'https://docs.lukso.tech/contracts/overview/LSP1Delegate',
+    urlABI: 'https://docs.lukso.tech/contracts/contracts/LSP6KeyManager',
+    description: (
+      <Translate id="playground.codesandbox.description">
+        Allows to react on transfers and informations received (token transfers,
+        notifications, etc...)
+      </Translate>
+    ),
+  },
+  {
+    name: '🌐 LSP3 Profile Metadata',
+    url: 'https://docs.lukso.tech/contracts/overview/UniversalProfile',
+    urlABI: '',
+    description: (
+      <Translate id="playground.codesandbox.description">
+        Add information to an account or a vault, such as a profile/cover image,
+        external links, attributes, etc...
+      </Translate>
+    ),
+  },
+  {
+    name: '🏦 LSP9 Vault',
+    url: '',
+    urlABI: 'https://docs.lukso.tech/contracts/contracts/LSP9Vault',
+    description: (
+      <Translate id="playground.codesandbox.description">
+        A digital vault that can hold and transfer any assets, for storing funds
+        separate from your main account.
+      </Translate>
+    ),
+  },
+  {
+    name: '🧩 LSP17 Contract Extension',
+    url: '',
+    urlABI:
+      'https://docs.lukso.tech/contracts/contracts/LSP17ContractExtension/LSP17Extendable',
+    description: (
+      <Translate id="playground.codesandbox.description">
+        Uses this to add more functionality to your contract, so that it can be
+        flexible and extended overtime.
+      </Translate>
+    ),
+  },
+  {
+    name: '🔄 LSP20 Call Verification',
+    url: '',
+    urlABI: 'https://docs.lukso.tech/contracts/contracts/LSP20CallVerification',
+    description: (
+      <Translate id="playground.codesandbox.description">
+        Module to make interacting with a smart contract easier, without having
+        to resolve through its owner first.
+      </Translate>
+    ),
+  },
+  {
+    name: '⛽ LSP25 Execute Relay Call',
+    url: '',
+    urlABI: 'https://docs.lukso.tech/contracts/contracts/LSP25ExecuteRelayCall',
+    description: (
+      <Translate id="playground.codesandbox.description">
```

Comment:
do you need translate here? it looks like a copy/paste mistake

Reply:
Yes it is